lib/internal/bootstrap/pre_execution.js

```diff
@@ -332,6 +335,61 @@ function initializeDeprecations() {
     enumerable: false,
     configurable: true
   });
+
+  let warnedForProto = false;
+  const ObjectPrototype = ObjectGetPrototypeOf({});
```

Comment:
Isn't this the same as `const { ObjectPrototype } = primordials` ?

Reply:
ah, it is